app/services/idv/vendor_validated.rb

```diff
@@ -1,21 +0,0 @@
-module Idv
-  module VendorValidated
-    extend ActiveSupport::Concern
-
-    def vendor_validate
```

Comment:
What I am seeing is that the parent `Step` class is calling things like `vendor_validator_class.new`, which would not work unless the child classes implement the `vendor_validator_class` method. 

I think it's helpful for the parent to some way implement these methods or `raise NotImplementedError` so it's easier for devs to know what they need to do in order to get a child step class to work. This is a common pattern, as explained here: https://www.safaribooksonline.com/library/view/ruby-cookbook/0596523696/ch08s14.html

On the other hand, I just saw this blog about about how we should let duck types be and not raise `NotImplementedError`: http://chrisstump.online/2016/03/23/stop-abusing-notimplementederror/

So I guess your call? I am used to the pattern of creating abstract classes this way, but perhaps it's time for this old 🐶  to learn some new tricks.